lib/streamlit/DeltaGenerator.py

```diff
@@ -1708,8 +1708,8 @@ def multiselect(
         -------
         >>> options = st.multiselect(
         ...     'What are your favorite colors',
-                ('Yellow', 'Red')
-        ...     ('Green', 'Yellow', 'Red', 'Blue'))
+                ('Green', 'Yellow', 'Red', 'Blue'),
```

Comment:
Since this is an inline docstring, you have to put the `...` in to show its a line continuation

Reply:
Good spot, ta - fixed.